node/resolvers/Mutations/Users.ts

```diff
@@ -412,7 +412,8 @@ const Users = {
         id,
         userId,
       })
-      .then((result: any) => {
+      .then(async (result: any) => {
+        await new Promise(events.sendEvent('', 'b2b-organizations-graphql.removeUser', { id, email }))
```

Comment:
To simplify, you should be able to do this:
```suggestion
        await events.sendEvent('', 'b2b-organizations-graphql.removeUser', { id, email })
```

Reply:
I did this before but for some reason it was not working until I put in a promise like this. I can try once more and I'll give a feedback

Reply:
There's also a lint error on this line, but making the above change will likely fix the lint error as well.

Reply:
@polishq I changed to the suggested solution and this time it worked. I've just pushed a new commit with the fix.